### PR TITLE
Add basic instructions on how to enable peril

### PR DESCRIPTION
**Pull request by rcmachado**
This PR adds basic instructions on how to enable peril in a Loadsmart repository.

I would like to include some screenshots, but I'm a little concerned about this being a public repository. Any thoughts on that @barbosa and @camilamaia?

Fixes \#15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@
 
 🔧 Settings for Loadsmart's hosted Danger instance
 
+## How to add Peril to my repo
+
+Go to [org settings][org-settings], and them select _Installed Github
```
Comment (barbosa) [inline README.md]: `them` -> `then`

File: README.md [hunk 2]
```diff
@@ -4,6 +4,15 @@
...
+Apps_.
+
+On the configuration screen for peril-loadsmart app, under _Repository
+Access_ section, choose the repository you want to peril check on and
+then just click _Save_ button.
```
Comment (barbosa) [inline README.md]: Maybe better? `choose the repository you want Peril to run and then click _Save_`

**Comment by rcmachado** [comment]
Thanks for the feedback, @barbosa. Can you please take a look again?

**Comment by rcmachado** [comment]
@camilamaia Good idea. Let me explore with that.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,29 @@
...
 🔧 Settings for Loadsmart's hosted Danger instance
 
+## How to add Peril to my repo
+
+**Note**: this applies to repositories under Loadsmart organization. If
+you want to install peril for your repositories, you probably want to
+take a look at the general instructions available at
+[peril repository][peril-repo].
+
+For Loadsmart organization, the app is already instaled and configured.
```
Comment (camilamaia) [inline README.md]: small typo: installed instead of instaled :)

File: README.md [hunk 4]
```diff
@@ -4,6 +4,29 @@
 
 🔧 Settings for Loadsmart's hosted Danger instance
 
+## How to add Peril to my repo
+
+**Note**: this applies to repositories under Loadsmart organization. If
+you want to install peril for your repositories, you probably want to
+take a look at the general instructions available at
+[peril repository][peril-repo].
```
Comment (camilamaia) [inline README.md]: Just to make sure, should this be a link?

File: README.md [hunk 5]
```diff
@@ -4,6 +4,29 @@
...
+
+For Loadsmart organization, the app is already instaled and configured.
+You'll only need to enable it for your repository.
+
+Go to [org settings][org-settings], and then select _Installed Github
+Apps_.
+
+Choose the one named `peril-loadsmart`:
+
+![peril-loadsmart app](images/choose-app.png)
```
Comment (camilamaia) [inline README.md]: I can't see the image, maybe it is just because github's preview is not able to show it.

File: README.md [hunk 6]
```diff
@@ -4,6 +4,29 @@
 
 🔧 Settings for Loadsmart's hosted Danger instance
 
+## How to add Peril to my repo
+
+**Note**: this applies to repositories under Loadsmart organization. If
```
Comment (camilamaia) [inline README.md]: 👏

File: README.md (hunk 5 above)
Comment (camilamaia) [reply README.md]: Never mind, it loaded now, sorry

Comment (rcmachado) [reply README.md]: If you go to the file on the branch, you can see it: https://github.com/loadsmart/peril-settings/blob/bf944ef9d444a4ddc8f9a36c3ef019ea13229114/README.md

File: README.md (hunk 4 above)
Comment (rcmachado) [reply README.md]: Yes! There was a typo in the identifier at the end of the file 🤦‍♂️

**Comment by rcmachado** [comment]
@barbosa @camilamaia I think the screenshots are kinda big. Let me reduce them also.

**Pull request merged**